<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>609</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>871</width>
    <height>609</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>871</width>
    <height>609</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="correct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>정답 0 / 10</string>
    </property>
   </widget>
   <widget class="QLabel" name="quizNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>문항 1 / 10</string>
    </property>
   </widget>
   <widget class="QLabel" name="quiz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>851</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>851</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>851</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>퀴즈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>문항</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>문항</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>문항</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ans4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>문항</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
